Assets/Scripts/Interface/Plans Monitor/Details/PlanDetailsTab.cs

```diff
@@ -15,16 +15,12 @@ public class PlanDetailsTab: MonoBehaviour
 	protected bool isActive { get; private set; }
 	protected virtual PlanDetails.EPlanDetailsTab tabType => PlanDetails.EPlanDetailsTab.Feedback;
 
-	private void Start()
-	{
-		Initialise();
-	}
-
-	protected virtual void Initialise()
+	public virtual void Initialise()
 	{
 		tabToggle.onValueChanged.AddListener(SetTabActive);
 		isActive = tabToggle.isOn;
 		tabContainer.SetActive(isActive);
+		Debug.LogWarning(gameObject.name + " set to activity: " + tabToggle.isOn);
```

Comment:
Is this logged message left from debugging or is it needed?
If it's not necessary I'd remove it. :)